frontend/src/Movie/Details/MovieDetails.js

```diff
@@ -173,6 +174,15 @@ class MovieDetails extends Component {
     this.setState({ titleWidth: width });
   }
 
+  onKeyUp = (event) => {
+    if (event.keyCode === 37) {
```

Comment:
Add to keyCodes and use `import * as keyCodes from 'Utilities/Constants/keyCodes';`